utilities/cwp/custom-war-packager.inc

```diff
@@ -0,0 +1,137 @@
+# function to download the CWP jar
+# $1: Working directory
+# $2: CWP version to use
+download_cwp() {
+
+	if [ "$#" -gt 1 ]
```

Comment:
tabs vs. spaces. If possible, it would be better to use spaces to follow the common Jenkins codestyle

Reply:
Since in make you cannot use spaces, I decided to use tabs everywhere. But I agree in change it for shell scripts.